### DISCUSSION TITLE
How can we improve?

### DISCUSSION THREAD

**Discussion by NS007-dev** [App-Choreography/OurContributors#28] [Team Posts]
######  [Our Website 🌐](https://app-choreography.github.io/) | [Repositories 📚](https://github.com/orgs/App-Choreography/repositories)

--------------------------------------------

###  🤔 *How do you think we can improve our organisation and aspire to become one of the best open source teams?* 🤔

**Fields to consider ✅ :** 

- How can we become more popular with contributors? 
- What are we missing?
- How can our repos become known?
- What new repositories would you like?

### Please comment below with ideas and we will take action! 

Thank you.

**Comment by thuongtruong1009** [comment]
I think we can make some repo with topics such as learning code, practice algorithms, or web design... Because they have any problems that need be solved by the cooperation of many people or the community.

**Reply by BaseMax** [reply]
Hi @thuongtruong1009, How it's going?

**Comment by NS007-dev** [comment]
@thuongtruong1009 Thank you for your ideas!

- We can create an app based project where it requires contributors to use web design, algorithms and code.
- I was thinking maybe a repo where there is a list of coding problems, and you have to solve them the quickest, and the most correct will move you up a leaderboard. There will be a range of coding language categories to choose from, and you will be added to the specific language leaderboard?

Maybe some competitive coding repositories, and also some collaborating coding repositories?
Let us know what you think!
Thank you.
